### PR TITLE
fix(argo-cd): Correct ApplicationSet controller port

### PR DESCRIPTION
**Pull request by visheyra**
This PR propose a fix for the #1199.

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [ ] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Review by mkilchhofer — CHANGES_REQUESTED** [review]
Hi @visheyra, thanks for your contribution.

These tasks still are open:
- Adapt Chart.yaml: bump version, and write a changelog line
- Sign-off your commit (git commit -s --amend)

For the change itself, i'd suggest to omit the containerPort from values and "hardcode" it as introduced here: https://github.com/argoproj/argo-helm/commit/e7a27468b235d867929dd921f58c5fa11fb3850c

It seems that @mikeeq missed that while he implemented the 2.3 changes

**Comment by visheyra** [comment]
@mkilchhofer Okay, I think indeed hardcoding it is better ^^. I pushed/force I think this cover all your points now ?

File: charts/argo-cd/templates/argocd-applicationset/service.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
   ports:
   - name: {{ .Values.applicationSet.service.portName }}
     port: {{ .Values.applicationSet.service.port }}
-    targetPort: {{ .Values.applicationSet.containerPort }}
+    targetPort: 7000
```
Comment (mkilchhofer) [inline charts/argo-cd/templates/argocd-applicationset/service.yaml:18]:
```suggestion
    targetPort: webhook
```
Let's use the named port - then its less hardcoding ;)

File: charts/argo-cd/Chart.yaml [hunk 2]
```diff
@@ -22,3 +22,4 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - "[Fixed]: Correct ArgoCD notification subscriptions type"
```
Comment (mkilchhofer) [inline charts/argo-cd/Chart.yaml:24]: The old line needs to be replaced/removed

**Review by mkilchhofer — CHANGES_REQUESTED** [review]

File: charts/argo-cd/Chart.yaml [hunk 3]
```diff
@@ -22,3 +22,4 @@ dependencies:
...
+    - "[Fixed]: Correct ArgoCD applicationset controller port"
```
Comment (mkilchhofer) [inline charts/argo-cd/Chart.yaml:25]:
```suggestion
    - "[Fixed]: Correct ArgoCD applicationset controller port"
```
The old line needs to be replaced/removed

**Review by mkilchhofer — APPROVED** [review]
Lgtm now 👍

**Pull request merged**